Backends/CLX-fb/mirror.lisp

```diff
@@ -21,7 +21,7 @@
 	    height h)
       (setf xlib-image (make-array (list height width)
                                    :element-type '(unsigned-byte 32)
-                                   :initial-element #x00FFFFFF))
+                                   :initial-element #xffffffff))
```

Comment:
shouldn't that be #ffffff00 ?

Reply:
If I understand correctly, X only uses the color channels, that is `(byte 0 24)` and the color is only visible if the backend fail to draw all pixels of the mirror. So maybe something recognizable like `#xc080e0a0` would be good? The screenshot shows the suggested color.
![screenshot9](https://user-images.githubusercontent.com/150189/88058328-9463f100-cb63-11ea-8655-6e26e2f890a5.png)